seed/migrations/0142_auto_20210614_2058.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.20 on 2021-06-14 20:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('seed', '0141_propertymeasure_useful_life'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='meter',
+            name='source',
+            field=models.IntegerField(choices=[(1, 'Portfolio Manager'), (2, 'GreenButton'), (3, 'BuildingSync'), (4, 'Portfolio Manager')], default=None, null=True),
+        ),
+        migrations.AlterField(
+            model_name='meter',
+            name='type',
+            field=models.IntegerField(choices=[(1, 'Coal (anthracite)'), (2, 'Coal (bituminous)'), (3, 'Coke'), (4, 'Diesel'), (5, 'District Chilled Water - Absorption'), (6, 'District Chilled Water - Electric'), (7, 'District Chilled Water - Engine'), (8, 'District Chilled Water - Other'), (9, 'District Hot Water'), (10, 'District Steam'), (11, 'Electric - Grid'), (12, 'Electric - Solar'), (13, 'Electric - Wind'), (14, 'Fuel Oil (No. 1)'), (15, 'Fuel Oil (No. 2)'), (16, 'Fuel Oil (No. 4)'), (13, 'Fuel Oil (No. 5 and No. 6)'), (18, 'Kerosene'), (19, 'Natural Gas'), (20, 'Other:'), (21, 'Propane'), (22, 'Wood'), (23, 'Cost'), (24, 'Electric - Unknown')], default=None, null=True),
```

Comment:
Added source type for PM Data Request (just called 'Portfolio manager' still)
Added meter type for Electricity - Unknown